database/postgres_store.go

```diff
@@ -30,19 +34,81 @@ func (d *postgresStore) Close() {
 	d.DB.Close()
 }
 
-func (d *postgresStore) SaveRequestEntry(entry RequestEntry) error {
-	query := `INSERT INTO rpc_endpoint_requests
-	(id, received_at, request_duration_ms, is_batch_request, num_request_in_batch, http_method, http_url, http_query_param, http_response_status, ip_hash, origin, host, error) VALUES (:id, :received_at, :request_duration_ms, :is_batch_request, :num_request_in_batch, :http_method, :http_url, :http_query_param, :http_response_status, :ip_hash, :origin, :host, :error)`
+func (d *postgresStore) SaveRequestEntries(entries []RequestEntry) error {
```

Comment:
I feel this function does too much magic. Please add comments and a function docstring.